contracts/core/extensions/CoreAccounting.sol

```diff
@@ -171,4 +164,63 @@ contract CoreAccounting is
             _quantity
         );
     }
+
+    /* ============ Internal Functions ============ */
+
+    /**
+     * Deposit any quantity of tokens into the vault.
+     *
+     * @param  _tokenAddress    The address of the ERC20 token
+     * @param  _quantity        The number of tokens to deposit
+     */
+    function depositInternal(
+        address _from,
+        address _to,
+        address _tokenAddress,
+        uint _quantity
+    )
+        public
```

Comment:
shouldn't this be `internal`?

Reply:
Yes...important catch. Imma push a fix immediately.